ebos/eclgenericproblem.hh

```diff
@@ -352,6 +353,7 @@ protected:
     std::vector<TabulatedTwoDFunction> rockCompTransMultWc_;
     std::vector<TabulatedFunction> rockCompPoroMult_;
     std::vector<TabulatedFunction> rockCompTransMult_;
+    std::vector<TabulatedFunction> permFactTransMult_;
 
```

Comment:
The permFactTransMult table is already in the Brine Module. Better to use that instead of creating a copy.